.circleci/config.yml

```diff
@@ -1,34 +1,22 @@
 version: 2
 
-# NOTE: We run these in CircleCI because it has better artifacts support.
-# TODO: Move to GitHub Actions when its artifacts support improves.
+# NOTE: We run these in CircleCI because it has better artifacts support
+# and it is possible to publish image diffs as HTML like pytest-mpl in the future.
 jobs:
 
   image-tests-mpl302:
     docker:
-      - image: astropy/image-tests-py37-mpl302:1.10
+      - image: circleci/python:3.8
```

Comment:
Maybe it's noise only as we're not dropping 3.8 that soon, but I suppose we could go straight to 3.9

Reply:
@astrofrog said on Slack that he wants to cross test with multiple Python versions. 🤷

Reply:
ah, ok, I missed that